src/utils/restClient.ts

```
@@ -7,38 +7,80 @@ interface RestClientParams {
 
 	return {
 		fetch,
 		get: async <Response = any, Params = Record<string, string>>(
 			endpoint: string,
 			params?: Params,
-			config?: AxiosRequestConfig
-		) => (await fetch.get<Response>(endpoint, { params, ...config }))?.data,
+			config?: Options & { baseURL?: string; responseType?: string }
+		) => {
+			const { baseURL: configBaseURL, responseType, ...kyOptions } = config || {};
+			const url = configBaseURL ? `${configBaseURL}/${endpoint}`.replace(/\/+/g, '/').replace(':/', '://') : endpoint;
```

ESLint

[error] Strings must use doublequote.

(@typescript-eslint/quotes)